### ISSUE TITLE
support os.Stat()

### ISSUE THREAD

**Issue by ripienaar** [d5/tengo#80]

**Comment by d5** [comment]
Agreed. 

I'd also like to add something like
- `os.read_file`: equivalent of Go's `ioutil.ReadFile`
- `os.write_file`: equivalent of Go's `iotuil.WriteFile`

**Comment by ripienaar** [comment]
yeah those two would be useful too, it seems a bit weird that some of these are insstances of file though like chdir and chown, like, how do you get a file instance without open? and there's no need to open them.  it's almost like we need a file module.

```
f := import("file")
stat := f.stat("some.file")
f.chmod("some.file", "0644")
bytes := f.readall("some.file")
fh := f.open("some.file")
```

this feels more natural than the current api to me

**Comment by d5** [comment]
I left a comment on your PR, but, I believe we can add `os.stat()` to match Go's [os.Stat](https://golang.org/pkg/os/#Stat) function.
